### PR TITLE
Add load test for ingestion-edge

### PR DESCRIPTION
**Pull request by relud**
cc @whd the load tests passed with actual pubsub and default configuration (except `LOG_LEVEL=WARNING`)

### CODE REVIEW

File: ingestion-edge/tests/load/test_throughput.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+
+from google.cloud.monitoring_v3 import MetricServiceClient
+from google.cloud.monitoring_v3.types import Aggregation, TimeInterval
+from typing import Any, Dict, Union
+import time
+
+stackdriver = MetricServiceClient()
```
Comment (jklukas) [inline ingestion-edge/tests/load/test_throughput.py]: Am I understanding correctly that this is using the stackdriver client to maintain metrics without ever sending to Stackdriver? That's neat.

Comment (relud) [reply ingestion-edge/tests/load/test_throughput.py]: I'm polling the built-in load balancer metrics metrics from stackdriver, so there's no custom metrics or anything for me to send to stackdriver. Their api is a bit weird in that `list_time_series` is how you poll metric values.
